pkg/cli/extension/endpoint/update.go

```diff
@@ -0,0 +1,69 @@
+package endpoint
+
+import (
+	"fmt"
+
+	"github.com/spf13/cobra"
+	"github.com/spf13/pflag"
+
+	"github.com/fuseml/fuseml-core/gen/extension"
+	"github.com/fuseml/fuseml-core/pkg/cli/client"
+	"github.com/fuseml/fuseml-core/pkg/cli/common"
+)
+
+type endpointUpdateOptions struct {
+	client.Clients
+	global       *common.GlobalOptions
+	endpointType string
+	config       common.KeyValueArgs
+}
+
+func newEndpointUpdateOptions(o *common.GlobalOptions) *endpointUpdateOptions {
+	return &endpointUpdateOptions{global: o}
+}
+
+func newSubCmdEndpointUpdate(gOpt *common.GlobalOptions) *cobra.Command {
+	o := newEndpointUpdateOptions(gOpt)
+	cmd := &cobra.Command{
+		Use:   `update [--type {internal|external}] [-c|--configuration KEY:VALUE]... {EXTENSION_ID} {SERVICE_ID} {ENDPOINT_URL}`,
+		Short: "Update the attributes of an existing FuseML extension endpoint",
```

Comment:
While we're are in grammar phase of a review ... isn't it rather _of existing FuseML extension endpoint_ ? Without the article here, but I'm not sure....

Reply:
it's either "of an existing endpoint" or "of existing endpoints", but both are correct